modules/virt-cnv-ssp-alerts.adoc

```diff
@@ -0,0 +1,132 @@
+[id="SSPFailingToReconcile_{context}"]
+== SSPFailingToReconcile alert
+
+.Description
+
+The SSP Operator’s pod is up, but the pod's reconcile cycle consistently fails. This failure includes failure to update the resources it is responsible for, failure to deploy the template validator, or failure to deploy or update the common templates.
+
+.Reason
+
+If the SSP Operator fails to reconcile, then the deployment of dependent components fails, reconciliation of component changes fails, or both. Additionally, the updates to the common templates and template validator reset and fail.
+
+.Troubleshoot
+
+* Check the ssp-operator pod's logs for errors:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
+
+oc -n $NAMESPACE describe pods -l control-plane=ssp-operator
+
+oc -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator
+----
+
+* Verify that the template validator is up. If the template validator is not up, then check the pod’s logs for errors.
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
+
+oc -n $NAMESPACE get pods -l name=virt-template-validator
+
+oc -n $NAMESPACE describe pods -l name=virt-template-validator
+
+oc -n $NAMESPACE logs --tail=-1 -l name=virt-template-validator
+----
+
+.Resolution
+
+Open a support issue and provide the information gathered in the troubleshooting process.
+
+//SSPOperatorDown Alert
+[id="SSPOperatorDown_{context}"]
+== SSPOperatorDown alert
+
+.Description
+
+The SSP Operator deploys and reconciles the common templates and the template validator.
+
+.Reason
+
+If the SSP Operator is down, then the deployment of dependent components fails, reconciliation of component changes fails, or both. Additionally, the updates to the common template and template validator reset and fail.
+
+.Troubleshoot
+
+* Check ssp-operator's pod namespace:
++
+[source,terminal]
+----
+export NAMESPACE="$(oc get deployment -A | grep ssp-operator | awk '{print $1}')"
+----
+
+* Verify that the ssp-operator’s pod is currently down.
++
+[source,terminal]
+----
+oc -n $NAMESPACE get pods -l control-plane=ssp-operator
+----
+
+* Check the ssp-operator's pod description and logs.
++
+[source,terminal]
+----
+oc -n $NAMESPACE describe pods -l control-plane=ssp-operator
+
+oc -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator
+----
+
+.Resolution
+
+* Open a support issue and provide the information gathered in the troubleshooting process.
+
+//SSPTemplateValidatorDown Alert
+[id="SSPTemplateValidatorDown_{context}"]
+== SSPTemplateValidatorDown alert
+
+.Description
+
+The template validator validates that virtual machines (VMs) do not violate their assigned templates.
+
+.Reason
+
+If every template validator pod is down, then the template validator fails to validate VMs against their assigned templates.
+
+.Troubleshoot
+
+* Check the virt-template-validator’s Operator pod namespace:
```

Comment:
Should this be `ssp-operator`?

Reply:
The upstream runbook source says `virt-template-validator`. But you may be correct.

@sradco Is the correct version `virt-template-validator` or `ssp-operator` in the SSPTemplateValidatorDown
 alert?

Reply:
@borod108 Can you please help here?

Reply:
@sradco Is there an answer for this question about `virt-template-validator` or `ssp-operator`?

Reply:
I don't know the answer to that.. @borod108 ?

Reply:
virt-template-validator - is a part of ssp-operator. For debugging this issue it would be useful to check both:

_ssp-operator pods_ and _virt-template-validator pods_